ui/src/authors/containers/AuthorPublicationsContainer.jsx

```diff
@@ -25,7 +24,6 @@ export const dispatchToProps = dispatch => ({
   onQueryChange(queryChange) {
     dispatch(fetchAuthorPublications(queryChange));
     dispatch(fetchAuthorPublicationsFacets(queryChange));
-    dispatch(fetchCitationSummary());
```

Comment:
how come this doesn't break the test :/ we have to have a look at that as well.

Reply:
It did break the test, I fixed the test